Jenkinsfile

```diff
@@ -26,6 +26,7 @@ pipeline {
 
       // ENVs for sanitycheck
       ARCH = "-a arm"
+      PLATFORM = "-p nrf9160_pca10090 -p nrf52_pca10040 -p nrf52840_pca10056"
```

Comment:
There are additional boards with upstream support:

```
$ ls -d boards/arm/nrf*pca*
boards/arm/nrf51_pca10028     boards/arm/nrf52811_pca10056  boards/arm/nrf52840_pca10059  boards/arm/nrf52_pca10040  boards/arm/nrf9160_pca10090
boards/arm/nrf52810_pca10040  boards/arm/nrf52840_pca10056  boards/arm/nrf52840_pca10090  boards/arm/nrf52_pca20020
```

Is there some way to use the same glob here?